setup.py

```diff
@@ -25,13 +26,19 @@
     else:
         macros = []
 
+    CFLAGS = ['-O2']
+    # CFLAGS = ['-g']
+    if platform.uname().system != 'Windows':
```

Comment:
`platform.platform()`

Reply:
platform.platform() gives long string with lots of additional information, not just system name

Reply:
Oh, I meant `platform.system()`

Reply:
Ok, thanks)